graphics/fontconfig/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup                   muniversal 1.0
 
 name                        fontconfig
 version                     2.15.0
-revision                    0
+revision                    1
```

Comment:
Python is only a build dependency in this port. Did the installed files change as a result of using a different python version? If not, the revision should not be increased.